psst/case/__init__.py

```diff
@@ -9,7 +9,7 @@
 import pandas as pd
 import numpy as np
 
-from .descriptors import Name, Version, BaseMVA, BusName, Bus, Branch, BranchName, Gen, GenName, GenCost, GenType, GenFuel, Load, Period, _Attributes
+from .descriptors import Name, Version, BaseMVA, BusName, Bus, BusString, Branch, BranchName, Gen, GenName, GenCost, GenType, GenFuel, Load, Period, _Attributes
```

Comment:
`BusString` doesn't appear to be used anywhere?

Reply:
see locations identified below